scenario/features/chaincodeevents.feature

```diff
@@ -53,3 +53,18 @@ Feature: Chaincode event listening
         And I set the transaction arguments to ["close", "after"]
         And I invoke the transaction
         Then I should receive a chaincode event named "close" with payload "after" on "listener2"
+
+    Scenario: Checkpoint of chaincode events
+        Given I create a checkpointer
+        And I use my checkpointer to listen for chaincode events from basic
+        And I prepare to submit an event transaction
+        And I set the transaction arguments to ["checkpoint", "one"]
+        And I invoke the transaction
+        Then I should receive a chaincode event named "checkpoint" with payload "one"
+        Then I should checkpoint the chaincode event
```

Comment:
Then steps are assertions and this is not an assertion. It should not be needed either. We already said in an earlier step that we were using a checkpointer to listen for events so checkpointing should happen (in the test implementation code) without an explicit checkpoint step